mvj/settings.py

```diff
@@ -89,6 +89,18 @@ def get_git_revision_hash():
 
 DATABASES = {"default": env.db()}
 
+if os.environ.get("GITHUB_WORKFLOW"):
```

Comment:
I think this is better to do to `mvj-ci.yml` with the environment variables. Then the GitHub Actions not requires any changes to code files. Check this: https://stackoverflow.com/questions/64126095/github-action-use-environment-variable-as-port

Reply:
I think it working something like this:
```
jobs:
  build:
    env:
        DATABASE_URL: 'postgis://postgres:postgres@localhost/github_actions'
...
    steps:
      - name: Check and run migrations
        env:
            DATABASE_URL:  ${{ env.DATABASE_URL }}
        run: | 
          python manage.py makemigrations --dry-run --check
          python manage.py migrate
       ...
```

Reply:
Yep, now that I think of it that should work. I'll try it with the `DATABASE_URL` environment variable.

Reply:
@tomijarvi Seems to work. I agree it's better this way, not having to touch the app settings. Thank you for this!

Reply:
Perfect!